packages/base64/package.json

```
@@ -33,7 +33,7 @@
   "scripts": {
     "build": "exit 0",
     "prepack": "tsc --build tsconfig.build.json",
-    "postpack": "git clean -f '*.d.ts*' '*.tsbuildinfo'",
+    "postpack": "git clean -fX \"*.d.ts*\" \"*.d.cts*\" \"*.d.mts*\" \"*.tsbuildinfo\"",
```

I suspect that this will work:
```suggestion
    "postpack": "git clean -fX \"*.d.{,c,m}ts*\" \"*.tsbuildinfo\"",
```

---

It does not

---

(likewise this syntax is unsupported by `gitignore`)

---

🐑,(bah)